Modules/Loadable/Markups/MRML/vtkMRMLMarkupsCurveNode.h

```diff
@@ -210,7 +210,7 @@ class  VTK_SLICER_MARKUPS_MODULE_MRML_EXPORT vtkMRMLMarkupsCurveNode : public vt
 protected:
   vtkSmartPointer<vtkCleanPolyData> CleanFilter;
   vtkSmartPointer<vtkTriangleFilter> TriangleFilter;
-  vtkSmartPointer<vtkTransformPolyDataFilter> SurfaceToWorldTransformer;
+  vtkSmartPointer<vtkTransformPolyDataFilter> SurfaceToLocalTransformer;
```

Comment:
Can you define somewhere what this local coordinate system is? How it is related to world